### PR TITLE
上传

### PR DESCRIPTION
**Pull request by 1943432515**

### CODE REVIEW

File: website/comment/models.py [hunk 1]
```diff
@@ -5,4 +5,6 @@ class comment(models.Model):
     content = models.TextField(verbose_name='评论内容')
     parent = models.ForeignKey('self', on_delete=models.CASCADE, null=True, verbose_name='父评论id')
     datetime = models.DateTimeField(auto_now_add=True, verbose_name='评论时间')
-
+    qq = models.IntegerField(verbose_name='QQ号')
+    email_field = models.CharField(verbose_name='邮箱')
```
Comment (litlighilit) [inline website/comment/models.py]: Why not name it as `email`?

File: website/comment/models.py [hunk 2]
```diff
@@ -5,4 +5,6 @@ class comment(models.Model):
...
+    api = models.CharField()
```
Comment (litlighilit) [inline website/comment/models.py]: what's this used for?

File: website/comment/models.py (hunk 1 above)
Comment (litlighilit) [reply website/comment/models.py]: `CharField` requires `max_length` as a parameter! (see the failed CI)

File: website/comment/models.py (hunk 2 above)
Comment (litlighilit) [reply website/comment/models.py]: This `CharField` lacks `max_length` too

**Review by litlighilit — CHANGES_REQUESTED** [review]

File: website/comment/models.py (hunk 2 above)
Comment (1943432515) [reply website/comment/models.py]: 这个参数我是看你文档上写的

File: website/comment/models.py [hunk 3]
```diff
@@ -5,4 +5,5 @@ class comment(models.Model):
     content = models.TextField(verbose_name='评论内容')
     parent = models.ForeignKey('self', on_delete=models.CASCADE, null=True, verbose_name='父评论id')
     datetime = models.DateTimeField(auto_now_add=True, verbose_name='评论时间')
-
+    qq = models.IntegerField(verbose_name='QQ号')
+    email = models.CharField(max_length=20, verbose_name='邮箱')
```
Comment (litlighilit) [inline website/comment/models.py]: Have you ever tested locally?
I mean try posting some data to your locally running developing server.

**Review by litlighilit — CHANGES_REQUESTED** [review]

File: website/comment/models.py (hunk 3 above)
Comment (litlighilit) [reply website/comment/models.py]: Then you'll get it that writing as this won't work as expected

File: website/comment/models.py [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
 
-
 class comment(models.Model):
-    content = models.TextField(verbose_name='评论内容')
-    parent = models.ForeignKey('self', on_delete=models.CASCADE, null=True, verbose_name='父评论id')
-    datetime = models.DateTimeField(auto_now_add=True, verbose_name='评论时间')
-
+    content = models.TextField(verbose_name='评论内容', default="null")
+    parent = models.ForeignKey('self', on_delete=models.CASCADE, null=True, verbose_name='父评论id', default="null")
+    datetime = models.DateTimeField(auto_now_add=True, verbose_name='评论时间', default="null")
```
Comment (litlighilit) [inline website/comment/models.py]: What the hell?
Why touch content, parent and datetime?

Comment (litlighilit) [reply website/comment/models.py]: None of them shall use "null" (a string!!!) as default!

**Review by litlighilit — CHANGES_REQUESTED** [review]
Hand-writing a blank value, like a 4-charaters length string "null", but putting aside an existing blank value (a.k.a. None) is seldom an appreciative style especially in languages that has notation of null/nil like Python, unless there's certain reasons to do so.

**Pull request merged**